frontend/server/src/Controllers/Contest.php

```diff
@@ -581,7 +582,7 @@ public static function getCommonDetails(
             );
         }
 
-        return [
+        $response = [
```

Comment:
nit:

```suggestion
        return [
```

esto estaba bien antes.

Reply:
Listo